content/_index.html

```diff
@@ -5,89 +5,176 @@
+            Kubeflow includes services to create and manage interactive <a target='_blank' class='text-light' href="https://jupyter.org/">Jupyter
+            notebooks</a>. You can choose custom images, and allocate specific
+            compute resources based on your data science needs. This
+            comprehensive experience allows you to create workflows locally, and
+            deploy them with ease.
+          </p>
+        </div>
+      </div>
+      <div class="card border-primary">
+        <img
+          src="logos/TensorFlow.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="TensorFlow"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">
+            TensorFlow model training
+          </h5>
+          <p class="card-text text-light">
+            Kubeflow has a built-in <a target='_blank' class='text-light' href="https://www.tensorflow.org/">TensorFlow </a> job operator that you can use to
+            train a machine learning model with TensorFlow. The training
+            controller can be configured to use various compute resources (CPUs
+            or GPUs) and is compatible with various cluster sizes.
+          </p>
+        </div>
+      </div>
+      <div class="card border-primary">
+        <img
+          src="logos/Models.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="Model Serving image"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">Model Serving</h5>
+          <p class="card-text text-light">
+            Kubeflow supports a <a target='_blank' class='text-light' href="https://www.tensorflow.org/">TensorFlow </a> Serving container to export trained
+            TensorFlow models to Kubernetes. Kubeflow is also integrated with
+            <a target='_blank' class='text-light' href="https://www.seldon.io/">Seldon Core <a/>, an open source platform for deploying machine learning
+            models on Kubernetes, and <a target='_blank' class='text-light' href="https://www.developer.nvidia.com/tensorrt/">NVIDIA TensorRT</a> Inference Server for
+            maximized GPU utilization when deploying ML/DL models at scale.
+          </p>
+        </div>
+      </div>
+    </div>
+    <br />
+    <div class="card-deck">
+      <div class="card border-primary">
+        <img
+          src="logos/Pipeline.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="Pipelines Logo"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">Pipelines</h5>
+          <p class="card-text text-light">
+			<a target='_blank' class='text-light' href="https://www.kubeflow.org/docs/pipelines/overview/pipelines-overview/" >Kubeflow’s pipeline dashboard</a> is a comprehensive solution to deploy
```

Comment:
<I'll come back to reviewing this card. I've run out of headspace for today :) but I want to get these first comments out for discussion.>

Reply:
I'm back! Thanks for including extra info in this text block, and for adding the link. I have just a couple of tweaks to suggest, resulting in this wording:

>Kubeflow Pipelines is a comprehensive solution for deploying and managing end-to-end ML workflows. Use Kubeflow Pipelines for rapid and reliable experimentation. You can schedule and compare runs, and examine detailed reports on each run.